diff --git a/upstream/ffx-1.1.4/sdk/src/backends/vk/ffx_vk.cpp b/upstream/ffx-1.1.4/sdk/src/backends/vk/ffx_vk.cpp
index b2439b6..c60f0c9 100644
--- a/upstream/ffx-1.1.4/sdk/src/backends/vk/ffx_vk.cpp
+++ b/upstream/ffx-1.1.4/sdk/src/backends/vk/ffx_vk.cpp
@@ -34,6 +34,7 @@
 #endif  // _WIN32
 
 #include <vulkan/vulkan.h>
+#include <mutex>
 
 // prototypes for functions in the interface
 FfxVersionNumber       GetSDKVersionVK(FfxInterface* backendInterface);
